receiver/awscontainerinsightreceiver/internal/k8sapiserver/k8sapiserver.go

```diff
@@ -217,26 +220,26 @@ func (k *K8sAPIServer) init() error {
 
 	clientSet := k.k8sClient.GetClientSet()
 	configMapInterface := clientSet.CoreV1().ConfigMaps(lockNamespace)
-	if configMap, err := configMapInterface.Get(ctx, lockName, metav1.GetOptions{}); configMap == nil || err != nil {
+	if configMap, err := configMapInterface.Get(ctx, k.leaderLockName, metav1.GetOptions{}); configMap == nil || err != nil {
```

Comment:
I'm guessing we aren't worried about the breaking change(removing default) because this is supported to be internal?

Reply:
The default is still maintained as `otel-container-insight-clusterleader` so when not specified, nothing will change for existing customers.
The default is specified here: https://github.com/amazon-contributing/opentelemetry-collector-contrib/pull/4/files#diff-20e55430a41b074dab0e6cc3ef2551146c6856cd76d2d3aff3f2450099acb0f4R53